<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="listview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1141</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>1131</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>ID </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PASSWORD </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2FA</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PROXY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NAME PAGE  </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TRẠNG THÁI </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ACTION </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ON/OFF  </string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="QuanLyTk">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Quản Lý Tài Khoản  </string>
      </property>
     </widget>
     <widget class="QPushButton" name="themtk">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Thêm </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;PASWORD&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;2FA &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fa">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stopall">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>60</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Dừng</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chayall">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>10</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Chạy Tất Cả </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ảnh&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;SỐ LẦN GỬI TIN NHẮN &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="comen">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>10</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string> Link Comment </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="delaybt">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>12</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>DELAY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kb">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Kịch Bản </string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendimg">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Gửi Ảnh </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CHẠY PAGE TỪ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cont_page1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ĐÊN&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="count_page2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Baidang">
     <widget class="QTableWidget" name="tabel_feed">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <column>
       <property name="text">
        <string>Nội Dung </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>File ẢNh</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Action</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_add_feed">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>1131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>Thêm Bài Đăng   </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_show_home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_feed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Quản Lý Bài Đăng</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
